frontend/pages/SoftwarePage/components/PackageForm/helpers.tsx

```diff
@@ -1,7 +1,8 @@
 import React from "react";
 
 import { IDropdownOption } from "interfaces/dropdownOption";
-import { ISoftwarePackage } from "interfaces/software";
+import { IAppStoreApp, ISoftwarePackage } from "interfaces/software";
+import { InstallType } from "pages/SoftwarePage/SoftwareAddPage/SoftwareFleetMaintained/FleetMaintainedAppDetailsPage/FleetAppDetailsForm/FleetAppDetailsForm";
```

Comment:
nit: Can we pull this reference up to a more global level since now FMA and custom packages support this and VPP will soon?

If we don't do it now we'll definitely want to when adding VPP automatic install, which IIRC includes swapping radio to checkbox on this.